doc/articles/platform-specific-xaml.md

```diff
@@ -133,3 +133,10 @@ More visually, platform support for the pre-defined prefixes is shown in the bel
 Where:
  * 'Win' represents Windows, and
  * 'Droid' represents Android
+
+### Cross target library
+Since version 3.0, the Uno platform prefixes have changed for cross targeted libraries. These libraries need to be compiled for every possible platform. For most platform, that does not change anything, but with the addition of Skia, the Uno platform cannot distinguish Skia and Wasm. 
+
+The prefix `netstdref` is available and will include the objects or properties in both Skia and Wasm build. A prefix `not_nestdref` can also be used to exclude them. Since Skia and Wasm are similar, it is often not necessary to make the distinction. 
```

Comment:
@alextrepa I think a little table (for `netstdref` and `not_nestdref`) like the previous one could be a great addition for this new part as it will give additional details for the correct namespace URL, mc:Ignorable, etc...